activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -581,6 +581,22 @@ def num_waiting_in_queue # :nodoc:
         @available.num_waiting
       end
 
+      # Return connection pool's usage statistic
+      # Example: <tt>{ max: 15, total: 1, busy: 1, dead: 0, idle: 0, num_waiting: 0, checkout_timeout: 5 }</tt>
+      def stat
+        synchronize do
+          {
+            max: size,
+            total: @connections.size,
```

Comment:
Great idea!  Personal preference but max and total are a bit confusing to me.  Naming is hard.  We instantiate a pool with a "size".  A pool contains "connections".  Therefore, my poor naming would be:

```
size: size
connections: @connections.size
```

Either way, this is helpful!